docs/source/markdown/podman-create.1.md

```diff
@@ -824,6 +824,12 @@ Specify the policy to select the seccomp profile. If set to *image*, Podman will
 
 Note that this feature is experimental and may change in the future.
 
+#### **--secret**=*secret*
+
+Give the container access to a secret.
+
+Secrets are managed using the `podman secret` command
```

Comment:
It would be good to have a line or two explaining what a secret is or does.
```suggestion
Secrets are managed using the `podman secret` command.
```

Reply:
and/or perhaps add a link that the bottom to the new secret create man page